### DISCUSSION TITLE
List of JSON strings into csv

### DISCUSSION THREAD

**Discussion by mrp100461** [Cinchoo/ChoETL#216] [Q&A]
How can I put many json files into one CSV.
my json is 
{
    "name": "Mark Pearson",
    "address": "Abc, england ",
     "Filename": "WMCA, MACA",
}

**Comment by Cinchoo** [comment]
open one csv writer, read each json file and write to it as below

```
using (var w = new ChoCSVWriter("output.csv"))
{
   using (var r1 = new ChoJSONWriter("file1.json"))
   {
      w.Writer(r1);
   }
   using (var r2 = new ChoJSONWriter("file2.json"))
   {
      w.Writer(r2);
   }
}
```

**Comment by mrp100461** [comment]
This will not do I need it to be a list of json strings,  so each string will be a new entry in the csv. I was going to use CHOETL.JSON bit 
some people are saying an FS is better.
